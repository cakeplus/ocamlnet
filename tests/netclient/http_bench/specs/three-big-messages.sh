. helpers.sh

start_test_server \
    -line 1 -file data/framed-big \
    -line 6 -file data/chunked-big \
    -line 11 -file data/framed-big
trap "stop_test_server" EXIT
request -get / -get / -get / -run
