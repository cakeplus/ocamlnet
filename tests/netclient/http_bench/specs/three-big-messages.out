6D5598EF597094624574D0554CCFD38C
EAFB6283D339C25DFE1FFD0CDABDB125
6D5598EF597094624574D0554CCFD38C
